.pre-commit-config.yaml

```diff
@@ -78,17 +78,3 @@ repos:
       - id: terraform_fmt
         args:
           - --args=-write=true
-
-  # markdown
-  - repo: https://github.com/executablebooks/mdformat
-    rev: 0.7.16
-    hooks:
-      - id: mdformat
-        files: ^docs/
```

Comment:
I'm a little confused about this one. Don't we want to auto-format all markdown files? As is, we only formatted the ones in the `docs/` directory and thus this PR removes this hook.

Reply:
Interesting. 👀 

Other than `docs/`, only `README.md` and `RELEASE.md` files need to be formatted in this repo. All the other markdown files are being synced from `nebari-dev/.github` and get formatted in the `.gtihub` repo.

I'm comfortable removing this bit, or if you'd like, we can explicitly mention `README.md` and `RELEASE.md` here.

Reply:
> only `README.md` and `RELEASE.md` files need to be formatted in this repo.

Oh, ok, I didn't know that.

> I'm comfortable removing this bit, or if you'd like, we can explicitly mention README.md and RELEASE.md here.

No strong opinion. If there were no issues in the past, I'm happy to drop the formatter for now. Just FYI: both files are not formatted according to this hook. This is what happened in https://github.com/nebari-dev/nebari/pull/1801/commits/52060831eb9b599cf8f66515666fdc36c76e141b#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5